public/nextjs_migration/client/js/analytics.js

```diff
@@ -26,8 +26,16 @@ async function init({ debugMode, ga4MeasurementId }) {
     })
 
     // Instrument CTA clicks for analytics.
-    document.querySelectorAll('[data-cta-id]').forEach(cta =>
-      cta.addEventListener('click', () => window.gtag('event', 'clicked_cta', { cta_id: cta.dataset.ctaId })))
+    document
+      .querySelectorAll("[data-cta-id]")
+      .forEach((cta) =>
+        cta.addEventListener("click", () =>
+          window.gtag("event", "clicked_cta", {
+            cta_id: cta.dataset.ctaId,
+            page_location: location.href,
```

Comment:
Also, on L23 above we used `window.location.hostname`. Should this be prefixed w/ `window.`?